sonic_y_cable/y_cable.py

```diff
@@ -92,9 +92,9 @@
 
 MAX_NUM_LANES = 4
 
-# definitions for enable/disable mode for autoswitch feature
-ENABLE_AUTO_SWITCH = 1
-DISABLE_AUTO_SWITCH = 0
+# definitions for switching modes inside muxcable
```

Comment:
Remove "definitions for"; unnecessarily verbose

Reply:
fixed